### ISSUE TITLE
Calibrate operation fails on Spectrum 71686

### ISSUE THREAD

**Issue by jywarren** [publiclab/spectral-workbench#338]
on https://spectralworkbench.org/spectrums/71686

The operation shows "could not be saved": https://publiclab.org/notes/mokeymark/02-17-2016/question-how-do-i-save-the-calibration-data#c13643

and a call to `/tags` returns a 500 error, with:

````
MultiJson::ParseError (399: unexpected token at '{"average":7.6499999999999995,"r":7.6499999999999995,"g":5.1000000000000005,"b":1'):
  app/models/spectrum.rb:513:in `latest_json_data'
  app/models/spectrum.rb:522:in `generate_hashed_values'
  app/models/spectrum.rb:502:in `generate_processed_spectrum'
  app/models/snapshot.rb:52:in `generate_processed_spectrum'
  app/models/spectrum.rb:319:in `add_snapshot'
  app/models/tag.rb:173:in `create_snapshot'
  app/controllers/tags_controller.rb:38:in `create'
````

I'm wondering if this is an issue of the blob being too long; could perhaps solve this by resolving #335.

**Comment by jywarren** [comment]
Notice that the calibration operation/tag is actually saved, but does not generate a snapshot.

**Comment by jywarren** [comment]
I'm going to guess that overall length is the reason: `spectrum.average.length` is 951, so, yikes, that's a lot of data.

**Comment by jywarren** [comment]
I believe that was correct, and the issue is resolved -- see this fork, which, since the 2.1.10 precision management changes, now calibrates properly: https://spectralworkbench.org/spectrums/72366
